### ISSUE TITLE
How can I test `changesInclude`?

### ISSUE THREAD

**Issue by leshik** [cirruslabs/cirrus-cli#433]
Can't figure out how to trigger it locally. Seems it always fails.

**Comment by edigaryev** [comment]
> Can't figure out how to trigger it locally. Seems it always fails.

Hello! Can you provide some configuration example that you're having issues with?

**Comment by leshik** [comment]
@edigaryev whatever, any simple task, like this:

```
task:
  only_if: "changesInclude('.cirrus.yml')"
  container:
    image: alpine
    cpu: 0.25
    memory: 256Mi
  script: echo Test!
```

How can I make it know that `.cirrus.yml` has changed? `cirrus run` does nothing.

**Comment by edigaryev** [comment]
A new flag is now available in [CLI version 0.58.0](https://github.com/cirruslabs/cirrus-cli/releases/tag/v0.58.0) that was just released:

```
--affected-files strings             comma-separated list of affected files for changesInclude function
```
